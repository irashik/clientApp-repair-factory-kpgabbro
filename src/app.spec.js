import React from 'react';
import { shallow, render, mount } from 'enzyme';
import renderer from 'react-test-renderer';
import App  from './app';

import Container from 'react-bootstrap/Container';
import Row from 'react-bootstrap/Row';
import { BrowserRouter as Router, 
          Route, 
          Switch,
       } from 'react-router-dom';


//       import { expect } from 'jest';


jest.mock();


jest.mock('./navbarpage.js');
jest.mock('./header.js')
jest.mock('./footer.js')


jest.mock('./home.js');
jest.mock('./inputDataSection/inputDataSection.js');
jest.mock('./reportEquipment.js');
jest.mock('./repairPlan/repairPlan.js');
jest.mock('./bidRequest/bidRequest.js');
jest.mock('./authpage.js');
jest.mock('./authRegister.js');
jest.mock('./userProfile');



describe("<App/>", () => {
    test.only('should render correctly', () => {

        const component = shallow(<App />)
        expect(component).toMatchSnapshot();

        



        
    });

    
});