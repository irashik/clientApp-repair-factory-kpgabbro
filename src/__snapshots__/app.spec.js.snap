// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> should render correctly 1`] = `
<Container
  className="container-fluid"
  fluid={true}
>
  <HeaderComponent />
  <NavbarPage />
  <BrowserRouter>
    <Routes>
      <Route
        element={<HomeComponent />}
        path="/"
      />
      <Route
        element={<UNDEFINED />}
        path="/inputData"
      />
      <Route
        element={<ReportEquipment />}
        path="/reportEquipment"
      />
      <Route
        element={<RepairPlan />}
        path="/repairPlan"
      />
      <Route
        element={<BidRequest />}
        path="/bidRequest"
      />
      <Route
        element={<RegisterComponent />}
        path="/register"
      />
      <Route
        element={<ProfileUserComponent />}
        path="/profile"
      />
      <Route
        element={<AuthComponent />}
        path="/auth"
      />
    </Routes>
  </BrowserRouter>
  <FooterComponent />
</Container>
`;
